sencha-workspace/SlateDemonstrationsTeacher/app/view/EditWindow.js

```diff
@@ -242,14 +274,15 @@ Ext.define('SlateDemonstrationsTeacher.view.EditWindow', {
             me.on('render', _fireEvent, me, { single: true });
         }
     },
-    
+
     initComponent: function() {
         var me = this,
             studentCombo;
-            
+
         me.callParent(arguments);
-        
-        studentCombo = Ext.ComponentQuery.query('combobox[reference=studentCombo]')[0];
```

Comment:
@aliyeysides FYI you should *never* be using Ext.ComponentQuery.query within your code (there are rare legit reasons to, but generally it should only be something you use from your console when debugging)

This code as it was would have grabbed any combobox off the screen (if multiple editwindows were open) or crashed if none were open. It doesn't matter since we're back to using the lookupReference feature of ViewControllers, but what should have been done before FYI was `this.down(`combobox[reference=studentCombo]')` so that the query would be scoped to the EditWindow at hand